pai-management/doc/hdfs.md

```diff
@@ -91,6 +91,93 @@ curl http://NAME_NODE_ADDRESS:50070/jmx
 Currently OpenPAI management tool doesn't deploy HDFS in a High Availability (HA) fashion. This will be added in a future release.
 For solution about the HA feature please refer [HDFS High Availability](https://hadoop.apache.org/docs/stable/hadoop-project-dist/hadoop-hdfs/HDFSHighAvailabilityWithNFS.html).
 
+# Access HDFS Data
+
+Data on HDFS can be accessed by various ways. Users can choose the proper way according to there needs.
+
+## HDFS Command
```

Comment:
should add table of contents to link to each items. As later this doc maybe linked by other docs